static-build/components/TestCard/styles.css

```
@@ -155,7 +155,7 @@
 .iconList figcaption {
   width: 0;
   height: 0;
-  text-indent: -99999999px;
+  overflow: hidden;
```

```suggestion
  overflow: hidden;
  overflow: clip;
```

@jakearchibald seems like a good place to use clip if available? https://drafts.csswg.org/css-overflow-3/#valdef-overflow-clip

---

If all browsers supported `clip` then yeah it's probably the right answer here, but I'm not worried about programmatic scrolling in this case. If we use `clip` here, we run the risk of breakages if initial implementations are buggy. We'd also have some browsers using 'hidden' and some using 'clip', which ups the complexity.

---

sgtm, then i think this PR is ready?